.travis.yml

```diff
@@ -62,14 +62,18 @@ script:
 #  - sct_check_dependencies
 # test functions & integrity
   - bash -c "sudo apt-get install -qq libalglib-dev libinsighttoolkit4-dev; apt-cache show libvtk6-dev && sudo apt-get install -qq libvtk6-dev; apt-cache show libvtk7-dev && sudo apt-get install -qq libvtk7-dev; cd $HOME/build/neuropoly/spinalcordtoolbox/dev/isct_propseg; mkdir -p build; cd build; cmake ..; make; echo"
-  - sct_testing -d 1
+  - pip install coverage
+  - bash -c 'echo -ne "import coverage\ncov = coverage.process_startup()\n" > sitecustomize.py'
+  - bash -c 'echo -ne "[run]\nconcurrency = multiprocessing\nparallel = True\n" > .coveragerc'
+  - COVERAGE_PROCESS_START="$PWD/.coveragerc" COVERAGE_FILE="$PWD/.coverage" PYTHONPATH="$PWD" sct_testing -d 1
+  - coverage combine
+  - ./unit_testing.sh
```

Comment:
I wonder if we shouldn't separate these two different tests. Because otherwise, we'll never know what lacks unit-testing or integration tests. However, I'm not sure if coveralls accepts two kinds of distinct testing report.

Reply:
IMHO:
- We'll know that we lack coverage, and improving it can be achieved either by unit or integration tests...
- Coveralls is nice and sexy but not as flexible as the bare ``coverage``; we can also get coverage reports locally.

Reply:
I mentioned having separate coverages because the integration tests are very different than the unit tests. In integration tests, we test with real data, while on unit testing we test with mocked data. But as you mentioned, we can have separate reports locally, so that's not an issue.